python/spear/config.py

```diff
@@ -37,6 +37,9 @@ def get_config(user_config_files):
     # at runtime, but isn't available when we are authoring our default_config.*.yaml files.
     config.URDFBOT.URDFBOT_PAWN.URDF_DIR = os.path.realpath(os.path.join(SPEAR_ROOT_DIR, "urdf"))
 
+    # If we do not set this here, yacs has issues merging a: {} type values.
```

Comment:
# Required to merge dictionaries (i.e., we may have a dictionary that is empty by default, and we would like to be able to override it with a non-empty dictionary from a user config file).

Reply:
Done